src/requirements/data/majors/fsad.ts

```diff
@@ -0,0 +1,219 @@
+    ],
+    fulfilledBy: 'credits',
+    perSlotMinCount: [3],
+  },
+  {
+    name: 'Psychology',
+    description: 'This fulfills the college distribution social sciences requirement.',
+    source: 'https://courses.cornell.edu/preview_program.php?catoid=45&poid=23626',
+    checker: includesWithSubRequirements(['HD 1150', 'HD 1170', 'PSYCH 1101']),
+    fulfilledBy: 'courses',
+    perSlotMinCount: [1],
+    slotNames: ['Course'],
+  },
+  {
+    name: 'Ethics-Sustainability',
+    description: 'Choose from one of the following.',
+    source: 'https://courses.cornell.edu/preview_program.php?catoid=45&poid=23626',
+    checker: includesWithSubRequirements([
+      'AEM 3205',
+      'BEE 3299',
+      'BSOC 2061',
+      'COMM 4300',
+      'CRP 3011',
+      'DEA 4220*',
+      'DSOC 3240',
+      'FSAD 3200',
+      'FSAD 4021',
+      'INFO 1200',
+      'INFO 4270',
+      'NTRES 3320',
+      'PHIL 2960',
+      'STS 2061',
+    ]),
+    fulfilledBy: 'courses',
+    perSlotMinCount: [1],
+    slotNames: ['Course'],
+  },
+  {
+    name: 'Introductory Economics',
+    description: 'This fulfills the college distribution social sciences requirement.',
+    source: 'https://courses.cornell.edu/preview_program.php?catoid=45&poid=23626',
+    checker: includesWithSubRequirements(['ECON 1110']),
+    fulfilledBy: 'courses',
+    perSlotMinCount: [1],
+    slotNames: ['Course'],
+  },
+  {
+    name: 'Fashion Design Core',
+    description:
+      'Students in the Fashion Design (Option I) major must complete the requirements below in addition to the Fashion Design and Management Common Requirements.',
+    source: 'https://courses.cornell.edu/preview_program.php?catoid=45&poid=23626',
+    checker: includesWithSubRequirements(
+      ['FSAD 1111'],
+      ['FSAD 1140'],
+      ['FSAD 1170'],
+      ['FSAD 1250'],
+      ['FSAD 1350'],
+      ['FSAD 1360'],
+      ['FSAD 1450'],
+      ['FSAD 2310'],
+      ['FSAD 2370'],
+      ['FSAD 2640'],
+      ['FSAD 2650'],
+      ['FSAD 2660'],
+      ['FSAD 3770'],
+      ['FSAD 4700'],
+      ['FSAD 4770']
+    ),
+    fulfilledBy: 'courses',
+    perSlotMinCount: [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
+    slotNames: [
+      'FSAD 1111',
+      'FSAD 1140',
+      'FSAD 1170',
+      'FSAD 1250',
+      'FSAD 1350',
+      'FSAD 1360',
+      'FSAD 1450',
+      'FSAD 2310',
+      'FSAD 2370',
+      'FSAD 2640',
+      'FSAD 2650',
+      'FSAD 2660',
+      'FSAD 3770',
+      'FSAD 4700',
+      'FSAD 4770',
+    ],
+  },
+  {
+    name: 'Fashion Design Additional Courses',
+    description:
+      'Take any two FSAD courses at the 3000, 4000, or 6000 level. Course work from Fashion Design Core Courses cannot count here. FSAD 4000, 4010, 4020, 4030, 4990 cannot count here.',
+    source: 'https://courses.cornell.edu/preview_program.php?catoid=45&poid=23626',
+    checker: [
+      (course: Course): boolean => {
+        const { subject, catalogNbr } = course;
+        return (
```

Comment:
adding to check that "FSAD 4000, 4010, 4020, 4030, 4990 cannot count here

here's a similar example in CS:
<img width="820" alt="Screen Shot 2022-04-01 at 1 54 00 AM" src="https://user-images.githubusercontent.com/14082828/161203499-8ea4a767-1c8f-4eb6-bc00-7eb8ed4f9e68.png">
"